server_addon/applications/server/settings.py

```diff
@@ -212,7 +212,13 @@ class ApplicationsAddonSettings(BaseSettingsModel):
         scope=["studio"]
     )
     only_available: bool = SettingsField(
-        True, title="Show only available applications")
+        True,
+        title="Show only available applications",
+        description="Enable to show only available applications in AYON Launcher"
```

Comment:
I would not mention AYON launcher as webactions are comming soon...
```suggestion
        description="Enable to show only available applications"
```